### ISSUE TITLE
Notification test assertNotified() error

### ISSUE THREAD

**Issue by TimeglitchD** [laravel-filament/filament#5820]
### Package

filament/notifications

### Package Version

v2.17.8

### Laravel Version

v10.1.5

### Livewire Version

v2.12.2

### PHP Version

PHP 8.1.0

### Problem description

While testing my notifications, the tests resulted in the following error: 

```
1) Tests\Unit\NotifyTest::test_Notify
Error: Class "Filament\Http\Livewire\Notifications" not found

/var/www/html/vendor/filament/notifications/src/Notification.php:238
/var/www/html/vendor/filament/notifications/src/Testing/TestsNotifications.php:20
/var/www/html/vendor/laravel/framework/src/Illuminate/Macroable/Traits/Macroable.php:124
/var/www/html/vendor/livewire/livewire/src/Testing/TestableLivewire.php:255
/var/www/html/tests/Unit/NotifyTest.php:25
/var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Testing/TestCase.php:173
```

### Expected behavior

When replacing the line 

`use Filament\Http\Livewire\Notifications; `

with

`use Filament\Notifications\Http\Livewire\Notifications;`

in Filament\Notifications\Notification, the tests run as expected:

```OK (1 test, 3 assertions)```

### Steps to reproduce

Create Livewire component Notify:
```
<?php

namespace App\Http\Livewire;

use Filament\Notifications\Notification;
use Livewire\Component;

class Notify extends Component
{
    public function render()
    {
        return view('livewire.notify');
    }

    public function save(){
        Notification::make()
        ->title('Saved successfully')
        ->send();
    }
}
```

Create test:
```
<?php

namespace Tests\Unit;

use App\Http\Livewire\Notify;
use Illuminate\Foundation\Testing\RefreshDatabase;
use Livewire\Livewire;
use Tests\TestCase;

class NotifyTest extends TestCase
{
    use RefreshDatabase;

    public function test_Notify()
    {
        Livewire::test(Notify::class)
            ->call('save')
            ->assertNotified(__('Saved successfully'));

    }
}
```

Run `sail phpunit --filter NotifyTest`

### Reproduction repository

https://github.com/TimeglitchD/filament-notification-testing-bug

**Comment by danharrin** [comment]
Haha this was actually fixed in #5819

**Comment by danharrin** [comment]
It'll be released next
